lib/main_containers.pm

```diff
@@ -55,10 +55,10 @@ sub load_image_tests_docker {
 }
 
 sub load_host_tests_podman {
-    unless (is_sle('<15-SP1')) {
+    if (is_leap('15.1+') || is_tumbleweed || is_sle("15-sp1+")) {
         # podman package is only available as of 15-SP1
         loadtest 'containers/podman';
-        loadtest 'containers/podman_image';
+        loadtest 'containers/podman_image' unless is_public_cloud();
```

Comment:
You don't need that condition here... main_publiccloud.pm has its own test loader for containers (which is a mistake IMO).